test/core/primitives/primitives_codec_test.cpp

```diff
@@ -347,53 +356,66 @@ TEST_F(Primitives, decodeTransactionValidityInvalid) {
   auto &&res = codec_->decodeTransactionValidity(stream);
   ASSERT_TRUE(res);
   TransactionValidity value = res.value();
-  kagome::visit_in_place(
-      value,                                       // value
-      [](Invalid &v) { ASSERT_EQ(v.error_, 1); },  // ok
-      [](Unknown &v) { FAIL(); },                  // fail
-      [](Valid &v) { FAIL(); });                   // fail
+  kagome::visit_in_place(value,                                       // value
+                         [](Invalid &v) { ASSERT_EQ(v.error_, 1); },  // ok
+                         [](Unknown &v) { FAIL(); },                  // fail
+                         [](Valid &v) { FAIL(); });                   // fail
 }
 
 /**
  * @given byte array containing encoded
  * and TransactionValidity instance as Valid
  * @when codec_->decodeBlockId() is applied
- * @thenobtained result matches predefined block id instance
+ * @then obtained result matches predefined block id instance
  */
 TEST_F(Primitives, decodeTransactionValidityUnknown) {
   Buffer bytes = {2, 2};
   auto stream = ByteArrayStream(bytes);
   auto &&res = codec_->decodeTransactionValidity(stream);
   ASSERT_TRUE(res);
   TransactionValidity value = res.value();
-  kagome::visit_in_place(
-      value,                                       // value
-      [](Invalid &v) { FAIL(); },                  // fail
-      [](Unknown &v) { ASSERT_EQ(v.error_, 2); },  // ok
-      [](Valid &v) { FAIL(); }                     // fail
+  kagome::visit_in_place(value,                                       // value
```

Comment:
What are these four comments here for?

Reply:
Let ask @masterjedy , he wrote that. I don't think comments are too helpful here, but they don't look ugly either